src/Wallabag/CoreBundle/Resources/views/themes/common/Entry/entries.xml.twig

```diff
@@ -1,34 +1,46 @@
 <?xml version="1.0" encoding="utf-8"?>
-<rss version="2.0" xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:media="http://search.yahoo.com/mrss/">
-    <channel>
-        <title>wallabag — {{type}} feed</title>
-        <link>{{ url(type) }}</link>
-        <link rel="self" href="{{ app.request.uri }}"/>
-        {% if entries.hasPreviousPage -%}
-            <link rel="previous" href="{{ url }}?page={{ entries.previousPage }}"/>
-        {% endif -%}
-        {% if entries.hasNextPage -%}
-            <link rel="next" href="{{ url }}?page={{ entries.nextPage }}"/>
-        {% endif -%}
-        <link rel="last" href="{{ url }}?page={{ entries.nbPages }}"/>
-        <pubDate>{{ "now"|date('D, d M Y H:i:s') }}</pubDate>
-        <generator>wallabag</generator>
-        <description>wallabag {{type}} elements</description>
-
-        {% for entry in entries %}
-
-            <item>
-                <title><![CDATA[{{ entry.title|e }}]]></title>
-                <source url="{{ url('view', { 'id': entry.id }) }}">wallabag</source>
-                <link>{{ entry.url }}</link>
-                <guid>{{ entry.url }}</guid>
-                <pubDate>{{ entry.createdAt|date('D, d M Y H:i:s') }}</pubDate>
-                <description>
-                    <![CDATA[{%- if entry.readingTime > 0 -%}{{ 'entry.list.reading_time_minutes'|trans({'%readingTime%': entry.readingTime}) }}{%- else -%}{{ 'entry.list.reading_time_less_one_minute'|trans|raw }}{%- endif %}{{ entry.content|raw -}}]]>
-                </description>
-            </item>
-
+<feed xmlns="http://www.w3.org/2005/Atom">
+    <title>wallabag — {{type}} feed</title>
+    <subtitle type="html">RSS feed for {{ type }} entries</subtitle>
+    {% if entries | length > 0 %}
+        <updated>{{ (entries | first).createdAt | date('c') }}</updated> {# Indicates the last time the feed was modified in a significant way. #}
+    {% endif %}
+    <id>wallabag:{{ domainName | removeScheme | removeWww }}:{{ user }}:{{ type }}</id>
+    <link rel="alternate" type="text/html" href="{{ url(type) }}"/>
```

Comment:
I'm wondering it `url(` generates an absolute url or not.
Otherwise use `{{ absolute_url(path(type)) }}`